rocks/RocksTypeDB.java

```diff
@@ -50,44 +44,17 @@
     }
 
     private final Options.Database typeDBOptions;
-    private final org.rocksdb.Options rocksDBOptions;
     private final RocksDatabaseManager databaseMgr;
     private final AtomicBoolean isOpen;
 
     protected RocksTypeDB(Options.Database options, Factory.DatabaseManager databaseMgrFactory) {
         if (!Executors.isInitialised()) Executors.initialise(MAX_THREADS);
         this.typeDBOptions = options;
-        this.rocksDBOptions = initRocksDBOptions();
         this.databaseMgr = databaseMgrFactory.databaseManager(this);
         this.databaseMgr.loadAll();
         this.isOpen = new AtomicBoolean(true);
     }
 
-    private org.rocksdb.Options initRocksDBOptions() {
```

Comment:
we no longer share Options between databases, and move their instantiation down to the `RocksDatabase` level